pkgs/development/python-modules/py-slvs/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  swig,
+  cmake,
+  ninja,
+  setuptools,
+  scikit-build,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "py-slvs";
+  version = "1.0.6";
+
+  src = python3Packages.fetchPypi {
+    pname = "py_slvs";
+    inherit version;
+    sha256 = "sha256-U6T/aXy0JTC1ptL5oBmch0ytSPmIkRA8XOi31NpArnI=";
+  };
+
+  pyproject = true;
+
+  nativeBuildInputs = with pkgs; [
+    swig
+  ];
+
+  build-system = [
+    cmake
+    ninja
+    setuptools
+    scikit-build
+  ];
+
+  dontUseCmakeConfigure = true;
+
+  meta = with lib; {
```

Comment:
we should probably have an imports check as well

Reply:
Just check that it's importable, not run full test cases, right?

Reply:
No, I'm referring to https://nixos.org/manual/nixpkgs/unstable/#using-pythonimportscheck here